platform/darwin/src/http_file_source.mm

```diff
@@ -83,8 +83,9 @@ void cancel() {
         @autoreleasepool {
             NSURLSessionConfiguration* sessionConfig =
                 [NSURLSessionConfiguration defaultSessionConfiguration];
-            sessionConfig.timeoutIntervalForResource = 30;
-            sessionConfig.HTTPMaximumConnectionsPerHost = 8;
+            //sessionConfig.timeoutIntervalForResource = 30;
+            sessionConfig.timeoutIntervalForRequest = 360;
+            sessionConfig.HTTPMaximumConnectionsPerHost = 30;
             sessionConfig.requestCachePolicy = NSURLRequestReloadIgnoringLocalCacheData;
```

Comment:
no, the swizzle is for the constructor, and then these values are reset after it's initialized.